### DISCUSSION TITLE
Restart Domain Transfer

### DISCUSSION THREAD

**Discussion by tobitaku** [vercel/community#5000] [Help]
### Summary

Hello,

I have recently started a domain transfer from GoDaddy into Vercel. An issue occurred during the transfer and now I need to restart the domain transfer within Vercel but haven't found a way to do so? I can restart the transfer on GoDaddy no problem.

Is there any way to abort/restart a pending domain transfer within Vercel? Please help!
Thanks

### Steps to Reproduce

- Start a domain transfer out of GoDaddy into Vercel and receive error
- Try to delete domain in Vercel
- Get error "Cannot delete <domain>. Transfer in progress"

**Comment by amyegan** [comment]
Hi Tobias. If you let me know the domain or project, I can check the logs for more info 🙏

**Reply by tobitaku** [reply]
Hi Amy, 
thanks for your reply. The account name is `tobitaku` and the project name is `tobiasziegler-dev`. Thank you for checking!

**Reply by amyegan** [reply]
Thanks for sharing that. The current status of the transfer says our system is waiting on the transfer to be approved by GoDaddy. Some registrars send an email for domain owners to approve or reject transfer requests, but transfers will typically be processed automatically within five days so long as the transfer is not rejected.

Do you have any more information from GoDaddy about the transfer approval or current status?

**Reply by tobitaku** [reply]
Hi Amy,

You are right - I just checked the transfer status with GoDaddy and it seems that the transfer was rejected by their side for some reason. However, I am able to restart the transfer process and get another verification code, but there is no way for me to enter the code since the transfer process is still pending within Vercel.

What do you think I should do next?

**Reply by amyegan** [reply]
I was able to cancel the pending transfer. Please try again and let me know if it works for you now 🙏
